pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -75,13 +81,15 @@ core = stdenv.mkDerivation rec {
 
   inherit (common) src prePatch;
 
-  outputs = [ "out" "doc" ];
+  outputs = [ "out" "doc" "crosstools" ];
```

Comment:
I think this should probably be `dev`, or at least one of the other standard output names.  There seems to be some consensus that we should use the standard ones except when none of them are even remotely applicable.

I don't know the entire reasoning for this, but @SuperSandro2000 probably does; he is the one who pointed this out to me.

Reply:
`dev` sounds reasonable enough to me, even though the output is only useful for cross-building texlive.bin (it generates one of the source files)